app/models/reports/repository.rb

```diff
@@ -100,15 +100,28 @@ def reporting_schema_v2?
 
     alias_method :adjusted_patients, :adjusted_patients_without_ltfu
 
-    # Return the running total of cumulative assigned patient counts.
+    # Returns cumulative assigned patients from facility_states - this includes LTFU
+    private def cumulative_assigned_patients_query_v2(region)
+      Reports::FacilityState.for_facility(region).order(:month_date).pluck(:month_date, :assigned_patients)
```

Comment:
I realized the better method name for this is `cumulative_assigned...`, to better match the conventions we have here in Repository. We probably want to change it in the matviews as well.

Reply:
Renamed in fe2501e6968a8b23864e88fedb3cf2d379bc17bb.